stdlib/2/typing.pyi

```diff
@@ -16,6 +16,7 @@ Type = object()
 _promote = object()
 no_type_check = object()
 ClassVar = object()
+Protocol = object()
```

Comment:
Yes, but if I write ``class Protocol: ...`` then mypy will _allow_ most things I mentioned above.
This is why ``object()`` is used also for ``Generic`` and friends. Otherwise prohibiting all possible invalid scenarios will require a lot of effort.